### DISCUSSION TITLE
How to make the JDTLS use class path?

### DISCUSSION THREAD

**Discussion by anshumankomawar** [mfussenegger/nvim-jdtls#70] [Q&A]
I have local jar files that I require as a dependency in my java project. I have added them to my classpath however when I open nvim I always get an error saying the import cannot be resolved. Is there a way for me to get this working? I don't have much experience working with jar files or eclipse so any help would be appreciated.

**Comment by mfussenegger** [comment]
How did you add the jar to your classpath? Is your project using maven or gradle?

Overall this sounds more like a general eclipse.jdt.ls related question and you might have more luck asking in https://github.com/eclipse/eclipse.jdt.ls
